### DISCUSSION TITLE
Is it possible to select random (float) numbers in a prompt?

### DISCUSSION THREAD

**Discussion by MancV21** [adieyal/sd-dynamic-prompts#550] [Q&A]
Hi. I'm using the wildcard __actress__ three times in my prompt, so that it generates random faces. I'd like to specify how much of each face to use using a float number. The normal prompt would be (Emilia Clarke:0.4) for example. Is it possible with this extension to create something like that?

(__actress__:randrange(0.1,1)) as an example way of doing it?

Thanks

**Comment by adieyal** [comment]
You could have a look at [Jinja templates](https://github.com/adieyal/sd-dynamic-prompts/blob/main/jinja2.md). Alternatively, An easy hack is to simply create a wildcard file with floating point numbers, e.g.
```
# random.txt
0
0.1
0.2
0.3
0.4
0.5
0.6
0.7
0.8
0.9
1.0
```

and then `__actress__:__random__`

I haven't tested it, but it should work.

**Reply by MancV21** [reply]
That last suggestion is genius! It's so simple but surely would work. Thank you!
